lib/pacemaker/pcmk_output.c

```diff
@@ -1999,6 +2019,21 @@ cluster_status_xml(pcmk__output_t *out, va_list args)
                      false);
     }
 
+
+    /* If there are verification errors, always print a statement about that, even if not requested */
+
+    pcmk__output_new(&verify_out, "none", NULL, NULL);
+    verify_rc = pcmk__verify(scheduler, verify_out, scheduler->input);
+    pcmk__output_free(verify_out);
+
+    if (verify_rc != pcmk_rc_schema_validation) {
+        if (pcmk_is_set(section_opts, pcmk_section_verify)) {
+            out->info(out, "No verification errors");
+        }
+    } else {
+        out->info(out, "Found verification errors! Run crm_verify for more information.");
+    }
```

Comment:
`out->info()` is ignored for XML output. This is where our work on crm_verify will come in handy. For XML, we don't need to create a "none" object, we can pass crm_mon's output object. That should end up with output something like this:

    <summary>
      <stack type="corosync"/>
      <current_dc present="true" version="" name="cluster02" id="2" with_quorum="true" mixed_version="false"/>
      <last_update time=""/>
      <last_change time=""/>
      <nodes_configured number="5"/>
      <verifications code="78" message="Invalid configuration">
        <errors>
          <error>Resource start-up disabled since no STONITH resources have been defined</error>
          <error>Either configure some or disable STONITH with the stonith-enabled option</error>
          <error>NOTE: Clusters with shared data need STONITH to ensure data integrity</error>
          <error>Node pcmk-1 is unclean but cannot be fenced</error>
          <error>Node pcmk-2 is unclean but cannot be fenced</error>
          <error>CIB did not pass schema validation</error>
          <error>Errors found during check: config not valid</error>
        </errors>
      </verifications>
      <resources_configured number="32" disabled="4" blocked="0"/>
      <cluster_options stonith-enabled="true" symmetric-cluster="true" no-quorum-policy="stop" maintenance-mode="false" stop-all-resources="false" stonith-timeout-ms="60000" priority-fencing-delay-ms="0"/>
    </summary>